android/app/src/main/res/layout/item_track.xml

```diff
@@ -1,43 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Remove this LinearLayout and make the CardView as the parent for this whole view.
Having an extra LinearLayout as parent makes no sense.